Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb_example.py

```diff
@@ -90,14 +95,21 @@ def meas():
 ##############################
 ##  Two-qubit RB execution  ##
 ##############################
-
+# create RB experiment from configuration and defined functions
 rb = TwoQubitRb(
-    config, bake_phased_xz, {"CZ": bake_cz}, prep, meas, verify_generation=False, interleaving_gate=None
-)  # create RB experiment from configuration and defined functions
+    config=config,
+    single_qubit_gate_generator=bake_phased_xz,
+    two_qubit_gate_generators={"CZ": bake_cz},  # can also provide e.g. "CNOT": bake_cnot
```

Comment:
Didn't you did it with cross resonance gates as well?

Reply:
Yes, but where CR was used to construct a CNOT. So far we don't support the CR on its own, but if we were to do that, then users would have a higher Clifford fidelity (not physical fidelity), since it costs less physical gates to implement each Clifford.